staging/src/k8s.io/apiserver/pkg/server/openapi/openapi_aggregator.go

```diff
@@ -0,0 +1,482 @@
+/*
```

Comment:
This file doesn't belong in the genericapiserver.  No "normal" apiserver has a reason to aggregate openapi specs, only the aggregator does that.  We should contain the logic where it is needed and keep the generic stuff separate.